tools/Status/Status.py

```diff
@@ -506,9 +553,35 @@ def render_status(
         "See `spectre -h` for its path."
     ),
 )
-def status_command(refresh_rate, **kwargs):
+@click.option(
+    "--columns",
+    "-c",
+    "input_columns",
+    callback=input_column_callback,
+    default=_default_columns_for_print(),
+    show_default=True,
+    help=(
+        "List of columns that will be printed for all jobs (executable specific"
+        " columns will be added in addition to this list). This can also be"
+        " specified in the config file with the name 'input_columns'. Note that"
+        " if the '--allusers'  option is not specified, then the \"User\""
+        " column will be omitted. Specify the columns as a comma separated list"
+        " wrapped in single quotes; e.g. 'Col1, Col2, Col3'"
```

Comment:
You don't need quotes if you remove the spaces (user should know that they need quotes if they put spaces on the command line)

Reply:
I'll add some docs explaining. I'll keep both possibilities though

Reply:
Ok. Use real col names I think.